src/main/java/commission/entity/Sale.java

```diff
@@ -1,9 +1,7 @@
 package commission.entity;
 
- public record Sale (long id, double price, double percentage, double commission) {
+import java.sql.Timestamp;
 
+public record Sale (long id, double price, Timestamp dateStamp, long personId) {
```

Comment:
Avoid using `java.sql.Timestamp` it's a database representation of a timestamp. This is used at a lower level (usually in the database driver). Rather use `java.time.Instant` for time values.